### PR TITLE
Prepare v0.60.0 release

### PR DESCRIPTION
**Pull request by ashutosh-narkar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,37 @@
...
+
+### Website + Ecosystem
+- Ecosystem: Digger ([#6464](https://github.com/open-policy-agent/opa/pull/6464)) authored by @anderseknert
+
+### Miscellaneous
+- ci: Fix the `Post-Merge` workflow by updating the token used in the checkout action step (authored by @ashutosh-narkar)
+- Update `Makefile` to allow custom `GOFLAGS` to be provided to the golang executable ([#6458](https://github.com/open-policy-agent/opa/issues/6458)) authored by @cova-fe
+- Dependency updates; notably:
+  - bump golang 1.21.4 -> 1.21.5 ([#6460](https://github.com/open-policy-agent/opa/pull/6460)) authored by @srenatus
+  - bump aquasecurity/trivy-action from 0.14.0 to 0.16.0
```
Comment (srenatus) [inline CHANGELOG.md:33]: Irrelevant, I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,37 @@
...
+
+### Docs
+- Add section on the changes proposed for a future OPA v1.0 and update Rego examples to be OPA v1.0 compliant([#6453](https://github.com/open-policy-agent/opa/issues/6453)) authored by @johanfylling
+- Clarify behavior of the `sprintf` builtin command when used with the `%T` marker ([#6487](https://github.com/open-policy-agent/opa/issues/6487)) authored by @lcarva
+
+### Website + Ecosystem
+- Ecosystem: Digger ([#6464](https://github.com/open-policy-agent/opa/pull/6464)) authored by @anderseknert
+
+### Miscellaneous
+- ci: Fix the `Post-Merge` workflow by updating the token used in the checkout action step (authored by @ashutosh-narkar)
```
Comment (srenatus) [inline CHANGELOG.md:29]: Worth including? Not sure 🤔

**Review by srenatus — DISMISSED** [review]
Thanks

**Review by johanfylling — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (ashutosh-narkar) [reply CHANGELOG.md:33]: We've included it before so I kept it.

File: CHANGELOG.md (hunk 2 above)
Comment (ashutosh-narkar) [reply CHANGELOG.md]: Removed.

**Pull request merged**